std/haxe/signals/SignalEmitter.hx

```diff
@@ -0,0 +1,3 @@
+package haxe.signals;
+
+interface SignalEmitter<T> extends Signal<T> extends Emitter<T> {}
```

Comment:
At first I was a bit confused about the name. `SignalEmitter` to me means the same thing as `Emitter`, i.e. "something that emits signals", not "`Signal` as well as `Emitter`".